internal/proxy/clientgo_cached_proxy_resolver_test.go

```diff
@@ -99,6 +100,29 @@ func Test_FetchCustomEntities(t *testing.T) {
 	}
 }
 
+func TestIsReady(t *testing.T) {
+	fakePostgresKong := sendconfig.Kong{InMemory: false}
+	fakeDblessKong := sendconfig.Kong{InMemory: true}
+	postgresProxy := clientgoCachedProxyResolver{
+		kongConfig: fakePostgresKong,
+		dbmode:     "postgres",
+	}
+	dblessProxy := clientgoCachedProxyResolver{
+		kongConfig: fakeDblessKong,
+		dbmode:     "off",
+	}
+
+	t.Log("checking initial readiness state")
+	assert.Equal(t, true, postgresProxy.IsReady())
```

Comment:
```suggestion
	assert.Equal(t, false, postgresProxy.IsReady())
```

Reply:
The `true` is intentional: https://github.com/Kong/kubernetes-ingress-controller/issues/1504#issuecomment-894503221

Database-backed instances don't need the controller to post configuration to obtain it, they can pull directly from the database. If the database isn't available, the Kong container will fail instead of the controller.